HousingRepairsOnlineApi.Tests/HelpersTests/RepairDayWindowHelperTests.cs

```diff
@@ -0,0 +1,110 @@
+            };
+            systemUnderTest = new RepairDayWindowHelper(repairDayWindows);
+        }
+
+        [Fact]
+#pragma warning disable CA1707
+        public void GivenRepairDayWindows_WhenCallingGetDaysForRepair_ReturnsCorrectDays()
+#pragma warning restore CA1707
+        {
+            // Arrange
+            _repair.Priority = "2";
+
+            // Act
+            var result = systemUnderTest.GetDaysForRepair(_repair);
+
+            // Assert
+            Assert.Equal(3, result);
+        }
+
+        [Fact]
+#pragma warning disable CA1707
+        public void GivenRepairDayWindows_WhenCallingGetDaysForRepairForSecondWindow_ReturnsCorrectDays()
+#pragma warning restore CA1707
+        {
+            // Arrange
+            _repair.Priority = "3";
+
+            // Act
+            var result = systemUnderTest.GetDaysForRepair(_repair);
+
+            // Assert
+            Assert.Equal(30, result);
+        }
+
+        [Fact]
+#pragma warning disable CA1707
+        public void GivenRepairDayWindows_WhenCallingGetDaysForRepairForSecondWindowWithManyResults_ReturnsFirstCorrectDays()
+#pragma warning restore CA1707
+        {
+            // Arrange
+            RepairDayWindowHelper repairDayWindowHelper = new RepairDayWindowHelper(new List<RepairDayWindow>()
+            {
+                new() { NumberOfDays = 3, Priority = "2" }, new() { NumberOfDays = 30, Priority = "3" }, new() { NumberOfDays = 3, Priority = "7" }
+            });
+            _repair.Priority = "2";
+
+            // Act
+            var result = repairDayWindowHelper.GetDaysForRepair(_repair);
+
+            // Assert
+            Assert.Equal(3, result);
+        }
+
+        [Fact]
+#pragma warning disable CA1707
+        public void GivenNoRepairDayWindows_WhenCallingGetDaysForRepairForSecondWindowWithManyResults_Returns0()
+#pragma warning restore CA1707
+        {
+            // Arrange
+            RepairDayWindowHelper repairDayWindowHelper = new RepairDayWindowHelper(new List<RepairDayWindow>());
+            _repair.Priority = "2";
+
+            // Act
+            var result = repairDayWindowHelper.GetDaysForRepair(_repair);
+
+            // Assert
+            Assert.Equal(0, result);
+        }
+
+        [Fact]
+#pragma warning disable CA1707
+        public void GivenNullRepair_WhenCallingGetDaysForRepair_throwsException()
+#pragma warning restore CA1707
+        {
+            // Arrange
+            RepairDayWindowHelper repairDayWindowHelper = new RepairDayWindowHelper(new List<RepairDayWindow>()
+            {
+                new() { NumberOfDays = 3, Priority = "2" }, new() { NumberOfDays = 30, Priority = "3" }, new() { NumberOfDays = 3, Priority = "7" }
+            });
+            _repair.Priority = "2";
+
+            // Act
+            Action act = () => _ = systemUnderTest.GetDaysForRepair(null);
+
+            // Assert
+            act.Should().ThrowExactly<ArgumentNullException>();
+        }
+    }
```

Comment:
Add a test for the behaviour when there are multiple `RepairDayWindow`s with the same `Priority` value.